tests/yamitranscode.cpp

```diff
@@ -62,6 +62,7 @@ static void print_help(const char* app)
     printf("   --btl1 <svc-t layer 1 bitrate: kbps > optional\n");
     printf("   --btl2 <svc-t layer 2 bitrate: kbps> optional\n");
     printf("   --btl3 <svc-t layer 3 bitrate: kbps> optional\n");
+    printf("   --enable-simulcast <enable h264 simulcast, priorityid also should be set)> optional\n");
```

Comment:
Can a new section be included with options for H.264 SVC?  So that the help messages is actually helpful.